firmware/library/L1_Drivers/test/can_test.cpp

```diff
@@ -1,5 +1,5 @@
 #include "L0_LowLevel/LPC40xx.h"
-#include "L1_Drivers/can.hpp"
+// #include "L1_Drivers/can.hpp"
```

Comment:
```suggestion
#include "L1_Drivers/can.hpp"
```

Reply:
Oh, I forgot to mention why I commented this out. I was getting undefined references to functions I'm using in queue.h. I'm assuming we need to mock this header somewhere?

Reply:
Hmm, looks like FakeIt needs an interface in order mock functions. Not sure how to handle this.